src/Lykke.Logs/Obsolete/LykkeLogToAzureBinder.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System;
 using AzureStorage.Tables;
 using Common.Log;
+using JetBrains.Annotations;
 using Lykke.SettingsReader;
 using Lykke.SlackNotifications;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace Lykke.Logs
 {
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
Fixed